src/message/create.ts

```diff
@@ -47,3 +56,19 @@ export function createNodesMessage(id: RequestId, total: number, enrs: ENR[]): I
     enrs,
   };
 }
+
+export function createTalkRequestMessage(request: string | Buffer, protocol: string): ITalkReqMessage {
```

Comment:
I think protocol should also be allowed to be a Uint8Array? Seems that the 'protocol' can be any bytestring.